docs/features/camera/orthographic-camera/orthographic-camera.md

```diff
@@ -0,0 +1,161 @@
+{
+    const float movementSpeed = 200;
+    _camera.Move(GetMovementDirection() * movementSpeed * gameTime.GetElapsedSeconds());
+}
+```
+
+You can now move the camera around using the Arrow keys.
+
+## Zooming with the camera
+
+The default value for zoom is 1.0f.  Increasing this value will make the scene larger, while decreasing will make it smaller.
+
+When possible try to use the convenient methods `ZoomIn` and `ZoomOut` instead of directly setting the `Zoom` value.  It's ok to set the `Zoom` value, but be aware `Zoom` can throw an exception if you pass too large of a value.  While the convenient methods clamp their values to prevent an error.
+
+```csharp
+// Add this to the Game1.cs file
+private void AdjustZoom()
+{
+    var state = Keyboard.GetState();
+    float zoomPerTick = 0.01f;
+    if (state.IsKeyDown(Keys.Z))
+    {
+        _camera.ZoomIn(zoomPerTick);
+    }
+    if (state.IsKeyDown(Keys.X))
+    {
+        _camera.ZoomOut(zoomPerTick);
+    }
+}
+
+// Add this to the Update() method
+AdjustZoom();
+```
+
+You can now press Z to zoom in, and X to zoom out.
+
+## Rotating the camera
+
+Rotating the camera is just as easy as zooming.  The important thing to be aware of is that the rotation is around the Center point called the `Origin`.  If you don't want to rotate the scene based on the middle of the screen, you'll need to adjust the origin.
+
+```csharp
+// Place this method in the Game1.cs file with the other methods
+private void RotateCamera()
+{
+    var state = Keyboard.GetState();
+    float rotateRadiansPerTick = 0.01f;
+    if (state.IsKeyDown(Keys.OemSemicolon))
+    {
+        _camera.Rotate(rotateRadiansPerTick);
+    }
+    if (state.IsKeyDown(Keys.OemQuotes))
+    {
+        _camera.Rotate(-rotateRadiansPerTick);
+    }
+}
+
+// Place this in the Update method
+RotateCamera();
+```
+
+You can press ; (semicolon) to rotate left, and ' (apostrophe) to rotate right.
+
+## Get World coordinates
+
+Last but not least, there'll be times when you want to convert from screen coordinates to world coordinates and vice-versa.  
+
+For example, if you want to know which sprite is under the mouse you'll need to convert the mouse position (screen position) back into the world position that was used to position the sprite in the first place.  Use the `ScreenToWorld` method for that.
+```csharp
+var mouseState = Mouse.GetState();
+_worldPosition = _camera.ScreenToWorld(new Vector2(mouseState.X, mouseState.Y));
+```
+
+If you need to instead convert a world position into a screen position, you can use `WorldToScreen`.  Assuming you have a character in the world called `player`, you can use this.
+```csharp
+_screenPosition = _camera.WorldToScreen(new Vector2(player.X, player.Y));
+```
+
+
+## Further Reading
+
+[Matrix Basics](https://stevehazen.wordpress.com/2010/02/15/matrix-basics-how-to-step-away-from-storing-an-orientation-as-3-angles/)
+[Orthographic Camera](https://en.wikipedia.org/wiki/Orthographic_projection)
```

Comment:
Change these to list items

Make this a markdown list item 

```md
- list item
- list item
```

Reply:
I've pushed up the fixes here also.